config/initializers/postgres_required_versions.rb

```diff
@@ -5,14 +5,13 @@ def initialize(*args)
   end
 
   def check_version
-    msg = "The version of PostgreSQL being connected to is incompatible with #{Vmdb::Appliance.PRODUCT_NAME} (10 required)"
+    msg = "The version of PostgreSQL being connected to (#{postgresql_version}) is incompatible with #{Vmdb::Appliance.PRODUCT_NAME} (130000 / 13 required)"
 
-    if postgresql_version < 90500
+    if postgresql_version < 10_00_00
```

Comment:
Note, I'm using the internal number convention that is in the pg adapter:

https://github.com/rails/rails/blob/7a6bcc6285a407a72933173f9e5aa610d9813337/activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb#L249-L266